db/migrate/20240312165451_rename_group_param_type.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class RenameGroupParamType < ActiveRecord::Migration[7.1]
+  def change
+    reversible do |dir|
+      dir.up do
+        Message.where(param_type: :group).find_each {|message| message.update(param_type: :group_requested) }
+      end
+
+      dir.down do
+        Message.where(param_type: :group_requested).find_each {|message| message.update(param_type: :group) }
```

Comment:
Just thinking aloud: Do we want to be notified about potential validation errors during migration? In this case, we would need to use `update!`. Otherwise, a `Message` currently stored in the database could be invalid (e.g., because the corresponding group was deleted and the message wasn't too), and update would just return `false` without failing the migration.